src/SFA.DAS.Commitments.Api.IntegrationTests/Helpers/ProbabilityDistribution.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.Commitments.Api.IntegrationTests.Helpers
+{
+    public class ProbabilityDistribution<T>
+    {
+        public class BoundaryValue
```

Comment:
should this class be in its own file? It is used as a normal class from the TestDataVolume class, but "appears" in this class to be an internal class?

Reply:
Good question! After some googling, esp. https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/nested-types, have extracted out the nested class and bound the two related classes together using the namespace instead.